core/src/test/java/org/mskcc/cbio/portal/scripts/TestImportStructuralVariantData.java

```diff
@@ -79,4 +82,59 @@ public void testImportStructuralVariantData() throws DaoException, IOException {
         assertEquals(0L, (long) structuralVariants.get(3).getSite2EntrezGeneId()); // null in db
         assertEquals(673L, (long) structuralVariants.get(3).getSite1EntrezGeneId());
     }
+
+    @Test
+    public void testImportStructuralVariantDataImportsCustomNamespacesFromTwoSamples() throws DaoException, IOException {
```

Comment:
Kan vervangen worden met een parametrized test.

Reply:
Fixed with @pvannierop in [4dd78fe](https://github.com/cBioPortal/cbioportal/pull/9880/commits/4dd78fe83c92add105ad383052636eacd650e55a)